tools/deploy_ipk.sh

```diff
@@ -21,33 +27,35 @@ deploy() {
     DEST_FOLDER=/tmp/prplmesh_ipks
 
     echo "Removing previous ipks"
-    ssh "$TARGET" "rm -rf \"$DEST_FOLDER\" ; mkdir -p \"$DEST_FOLDER\""
+    eval ssh "$SSH_OPTIONS" "$TARGET" \""rm -rf $DEST_FOLDER ; mkdir -p $DEST_FOLDER\""
```

Comment:
You removed the quoting around the folder names, so if they contain spaces, stars, or other special characters things will break.
Also since you added eval, you probably need another layer of quoting for SSH_OPTIONS and TARGET as well to get the same behavior as before (not using eval is easier!).

Since we probably won't use spaces in folder names, I let you decide if you want to fix it or not.

Reply:
I really prefer not to use `eval` but i'm not sure how, see my comment in https://github.com/prplfoundation/prplMesh/pull/676#discussion_r378348035

Reply:
Brought the quoting back